dev-client/src/screens/SoilScreen/ColorScreen/ColorScreen.tsx

```diff
@@ -159,7 +82,9 @@ export const ColorScreen = (props: SoilPitInputScreenProps) => {
 
   const color: MunsellColor | undefined = useMemo(
     () =>
-      data?.colorHue && data?.colorChroma && data?.colorValue
```

Comment:
suggestion: Is this logic something we could extract to the utils module? It seems like a reasonably standalone bit of logic that could be tested or might be useful somewhere else.